app/views/sessions/new.html.erb

```diff
@@ -11,7 +11,7 @@
           </div>
           <div class="form-group row">
             <%= f.label :password, class: 'col-sm-4' %>
-            <%= f.text_field :password, class: 'col-sm-8' %>
+            <%= f.password_field :password, class: 'col-sm-8' %>
```

Comment:
Re: Greg's comment. This bit should be squashed into the commit where the form fields are introduced.

Try as much as possible not to fix commits with more commits within the same PR. Before it is merged you have full control of changing every commit within the PR, so every commit should be as pristine as you can make it. 

After a PR is merged into master, we mostly have no choice but to fix problems with additional commits. So its best to fix any bad commit history before things are merged when you have control over it.